### ISSUE TITLE
YouTube and MOD issue

### ISSUE THREAD

**Issue by jackaco2** [GoelBiju/pinybot#34]
Hi guys, very sorry if this has been covered before but how do I go about assigning users as temporary mods and allowing anyone to queue a youtube video? 

Also is there a link to the bot commands anywhere? Thanks.

**Comment by Technetium1** [comment]
@jackaco2 Botters are able to play YouTube, as well as do a lot of other things. The [Features](https://github.com/GoelBiju/pinybot/wiki/Features) page of the Wiki has all of the commands.

**Comment by ashleymeah** [comment]
I got into this project because of a friend who has a tinychat room - I showed him how to change botter to "temp mod", and uses the command tmod. Of course this is all down to personal preferences but the other bot he knows of, uses different commands. Example !q instead of !yt.

I see why and understand your route - but there will likely be more discussion about this topic if this is not their first bot.
